resources/resource_ids.spec

```diff
@@ -1,223 +1,225 @@
     "META": {"sizes": {"includes": [20]}},
-    "includes": [30300]
+    "includes": [32320]
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-ai_chat_ui/ai_chat_ui.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30320],
+    "includes": [32340],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-skus_internals/skus_internals.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30340],
+    "includes": [32360],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-brave_news_internals/brave_news_internals.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30360],
+    "includes": [32380],
   },
   # This file is generated during the build.
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-line_chart_display/line_chart_display.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30380]
+    "includes": [32400]
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-ai_rewriter_ui/ai_rewriter_ui.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30400],
+    "includes": [32420],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-rewards_page/rewards_page.grd": {
     "META": {"sizes": {"includes": [50]}},
-    "includes": [30420],
+    "includes": [32440],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-creator_detection/creator_detection.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30440],
+    "includes": [32460],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-ads_internals/ads_internals.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30460],
+    "includes": [32480],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-custom_site_distiller_scripts/custom_site_distiller_scripts.grd": {
     "META": {"sizes": {"includes": [50]}},
-    "includes": [30480],
+    "includes": [32500],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/browser/resources/brave_education/resources.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30500],
+    "includes": [32520],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-new_tab_takeover/new_tab_takeover.grd": {
     "META": {"sizes": {"includes": [10]}},
-    "includes": [30520],
+    "includes": [32540],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-brave_new_tab_page_refresh/brave_new_tab_page_refresh.grd": {
     "META": {"sizes": {"includes": [30]}},
-    "includes": [30540],
+    "includes": [32560],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-email_aliases/email_aliases.grd": {
     "META": {"sizes": {"includes": [40]}},
-    "includes": [30560],
+    "includes": [32580],
   },
   "<(SHARED_INTERMEDIATE_DIR)/brave/components/brave_account/resources/resources.grd": {
     "META": {"sizes": {"includes": [25]}},
-    "includes": [30580],
+    "includes": [32600],
   },
-  # WARNING: The upstream ChromeOS/Ash strings currently run through 36930. We
+  # WARNING: The upstream ChromeOS/Ash strings currently run through 40000. We

```

Comment:
I don't think we actually care about this, we will probably never have BraveOS

Reply:
Right, we don't care about that, it's just that the upstream strings range for ChromeOS is between 31600 and 40000. I guess ultimately it doesn't matter since the ids that follow would just get pushed out until we hit the 2^16 limit again.